packages/components/src/tools-panel/test/index.tsx

```diff
@@ -1126,9 +1160,13 @@ describe( 'ToolsPanel', () => {
 			expect( announcement ).toHaveAttribute( 'aria-live', 'assertive' );
 
 			const disabledResetAllItem = await screen.findByRole( 'menuitem', {
-				disabled: true,
+				name: 'Reset all',
 			} );
 			expect( disabledResetAllItem ).toBeInTheDocument();
+			expect( disabledResetAllItem ).toHaveAttribute(
+				'aria-disabled',
+				'true'
+			);
```

Comment:
Any reason to not use `.toBeDisabled()` here?

Reply:
Unfortunately `toBeDisabled` doesn't take the `aria-disabled` attribute into account (see [the docs](https://github.com/testing-library/jest-dom#tobedisabled) and https://github.com/testing-library/jest-dom/issues/144)